package/batocera/core/batocera-configgen/configgen/configgen/generators/odcommander/odcommanderGenerator.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+from generators.Generator import Generator
+import Command
+import controllersConfig
+
+class OdcommanderGenerator(Generator):
+
+    def generate(self, system, rom, playersControllers, gameResolution):
+        commandArray = ["od-commander"]
+
+        # pad number
+        nplayer = 1
+        for playercontroller, pad in sorted(playersControllers.items()):
+            if nplayer == 1:
+                commandArray.append("joynum={}".format(pad.index))
+            nplayer += 1
```

Comment:
The `od-commander` binary doesn't know what to do with the `joynum` argument, and it's probably fine for the UI to be controllable from any connected controller anyway?

```suggestion
```

Reply:
Fair point.